app/config/services.php

```diff
@@ -59,18 +168,38 @@
     return $view;
 }, true);
 
+
 /**
- * Database connection is created based in the parameters defined in the configuration file
+ * Flash service with custom CSS classes
  */
-$di->set('db', function () use ($config) {
-    return new DbAdapter(array(
-        'host' => $config->database->host,
-        'username' => $config->database->username,
-        'password' => $config->database->password,
-        'dbname' => $config->database->dbname
-    ));
+$di->set('flash', function () {
+    $flash = new Session(
+        array(
+            'error' => 'alert alert-danger',
+            'success' => 'alert alert-success',
+            'notice' => 'alert alert-info',
+            'warning' => 'alert alert-warning'
+        ));
+
+    return $flash;
 });
 
+
+/**
+ * Database connection is created based in the parameters defined in the configuration file
+ */
+$di->set('db', function () use ($di) {
+    return new DbAdapter([
```

Comment:
I've added $di->get('config')->database->adapter to the config.dist.php and config.php files, but I've put this on a 'todo' item. You have to 'use' another line in the use clause (top of the page) and then call the adapter from your $di->set('db') clause. It's more complicated than just using 'adapter'. Config files are prepared.